components/pages/Pages.stories.tsx

```diff
@@ -1,9 +1,9 @@
 import { Meta } from "@storybook/react/types-6-0";
-import IndexPage from "./Index";
+import Home from "../../pages";
```

Comment:
Is this import really working?
```suggestion
import Home from "../../pages/index";
```